test/index.test.ts

```diff
@@ -6,11 +6,19 @@ Object.defineProperty(window.document, "referrer", {
   value: origin,
   writable: true,
 });
+
+Object.defineProperty(window, "location", {
+  value: {
+    href: origin + "?id=appid",
```

Comment:
we could even provide it with a domain (meaning
```
href: origin + "?domain=saleor.domain&id=appid",
```

Reply:
I will also allow us to test `createApp` without `domain` provided (which we don't have covered right now and actually is only recognised way used in wild).